README.md

```diff
@@ -158,43 +158,46 @@ router Docker image is set to the image built and push from the previous step.
 
 ```yaml
 # config-dev.yaml
-BatchRunnerConfig:
-  TimeInterval: 10s
+BatchEnsemblerConfig:
+  Enabled: true
+  JobConfig:
+    DefaultEnvironment: dev
+    DefaultConfigurations:
+      SparkConfigAnnotations:
+        "spark/spark.sql.execution.arrow.pyspark.enabled": "true"
+      BatchEnsemblingJobResources:
+        DriverCPURequest: "1"
+        DriverMemoryRequest: "1Gi"
+        ExecutorReplica: 2
+        ExecutorCPURequest: "1"
+        ExecutorMemoryRequest: "1Gi"
+  RunnerConfig:
+    TimeInterval: 10s
+    RecordsToProcessInOneIteration: 10
+    MaxRetryCount: 3
+  ImageBuildingConfig:
+    ImageConfig:
+      Registry: ghcr.io
```

Comment:
Would be nice to reflect in the config key name, that this is the destination registry